gnd/src/main/res/layout/text_input_field.xml

```diff
@@ -1,5 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
 <!--
-  ~ Copyright 2018 Google LLC
+  ~ Copyright 2019 Google LLC
```

Comment:
We shouldn't update old copyright dates in existing files. Could you kindly revert these changes and allow any year in the check?

Reply:
It does allow any year. (See the template xml.header, it uses regex <any 4 letter digit>)

I updated the year because I had to add the xml tag at the top

Reply:
Java
------
```
/*
 * Copyright <year> <any non-blank string>
 *
 * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
 * in compliance with the License. You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under the License
 * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 * or implied. See the License for the specific language governing permissions and limitations under
 * the License.
 */
```

XML
------
```
<?xml version="1.0" encoding="utf-8"?>
<!-- Copyright <year> <any non-blank string>

 Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
 in compliance with the License. You may obtain a copy of the License at

 http://www.apache.org/licenses/LICENSE-2.0

 Unless required by applicable law or agreed to in writing, software distributed under the License
 is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 or implied. See the License for the specific language governing permissions and limitations under
 the License.
--> 
```

Reply:
Thanks for that. Iiuc the actual Copyright line is optional for Google employee's shall we remove that from the regex (i.e. make it optional)?

Reply:
Are you referring to the below line?
`Copyright 2018 Google LLC`

Reply:
Actually, please disregard, better we're checking. Thanks!